Makefile

```diff
@@ -24,6 +24,15 @@ debug: $(TIMESTAMP)
 calibrate: $(TIMESTAMP)
 	$(ENV_PYTHON) calibrate.py
 
+check: $(TIMESTAMP)
+	$(ENV_PYTHON) check.py
+
+install-udev: 60-tycho.rules
+	sudo cp $< /etc/udev/rules.d/
+
```

Comment:
Worth adding a reload after copying the udev rules for convenience?
`sudo udevadm control --reload-rules && sudo udevadm trigger`